builder/alicloud/ecs/builder.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"github.com/hashicorp/hcl/v2/hcldec"
 	"github.com/hashicorp/packer/common"
+	"github.com/hashicorp/packer/common/commonsteps"
```

Comment:
Love that name !

Reply:
I like the idea of moving this into the multistep pkg once ready. 

With that said for the new plugin directory structure what do you think about dropping the common parent pkg and just having everything that is currently slated to be under `common` just at the top level of the SDK? Since most of the SDK will be common pkgs.  This comes to mind as it was something mentioned in our Go Style docs https://eng-handbook.hashicorp.services/language-style/go/#naming

Reply:
Good thinking -- that change is in PR plugin-sdk-6 IIRC